psydac/linalg/stencil.py

```diff
@@ -980,6 +1055,38 @@ def _tocoo_parallel_with_pads( self ):
 
         return M
 
+    # ...
+    def topetsc( self ):
```

Comment:
This function needs a docstring.

As I mentioned before, we should really handle the serial case as well.